week_1/__tests__/Array.test.js

```diff
@@ -28,3 +33,35 @@ test("max_subarray works", () => {
 test("two_sum works", () => {
 	expect(twoSum([2, 7, 11, 15], 9)).toEqual([0, 1]);
 });
+
+test("majority_element works", () => {
+	expect(majorityElement([3, 2, 3])).toBe(3);
+});
+
+test("merge_two_sorted_lists works", () => {
+	expect(mergeSortedArrays([1, 4, 6, 7, 9], [1, 3, 5, 5, 7, 8, 9, 20])).toEqual(
+		[1, 1, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9, 20]
+	);
+});
+
+test("count_unique_values works", () => {
+	expect(countUniqueValues([[1, 2, 3, 4, 4, 4, 7, 7, 12, 12, 13]])).toEqual(7);
+});
+
+test("max_subarray_sum works", () => {
```

Comment:
instead of "works", describe what "works" means

"max_subarray_sum can take in an array of numbers and return 'something'

This helps a success or failure reflect a specific piece of something succeeding or failing. It might be easier to picture each of these tests as a part of a broader application, maybe as components in a React app. You to use the test name a descriptor for an action that you expect to succeed. "my_function will return an error if given an array of strings", "my_function will sum up all of the _insert_condition_ and return the summed values", etc